### PR TITLE
Version 1.32.0 → 1.33.0

### PR DESCRIPTION
**Pull request by Gabriel439**

### CODE REVIEW

File: dhall-yaml/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+1.1.1
```
Comment (sjakobi) [inline dhall-yaml/CHANGELOG.md:1]: `dhall-yaml` exposes the `Options` type too.

```suggestion
1.2.0
```

File: dhall-yaml/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+
+* [Add `--generated-comment` flag for `dhall-to-yaml{-ng}`](https://github.com/dhall-lang/dhall-haskell/pull/1840)
+    * You can now optionally add a comment header to the YAML output
+      indicating that the file is generated and should not be hand-edited
```
Comment (sjakobi) [inline dhall-yaml/CHANGELOG.md:5]: I think this should be duplicated to the `dhall-json` changelog where `dhall-to-yaml` lives.

(Maybe we should discuss consolidating the `dhall-to-yaml{-ng}` situation…)

File: dhall-yaml/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+* [Produce output compatible with YAML 1.1](https://github.com/dhall-lang/dhall-haskell/pull/1788)
+    * Special strings like `on` are now quoted in order to avoid being
+      misinterpreted as boolean values by YAML 1.1 implementations
```
Comment (sjakobi) [inline dhall-yaml/CHANGELOG.md:8]: Same here, _I think_?!

File: dhall/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,40 @@
...
+    * If you really don't want to handle the new error-related wrapper, you can
+      get the old behavior using a partial pattern match (which will be partial,
+      still an improvement over the previous behavior, which was hanging)
+* [Fix invalid cache entries](https://github.com/dhall-lang/dhall-haskell/pull/1793)
+    * The interpreter will now correct cached expressions that are incorrect
+      and warn you when this happens
+    * Specifically, if there is a hash mismatch from the cached expression the
+      interpreter will resolve the import again and fix the cache if the
+      resolved import matches the expected hash
+* [Make `encodeExpression` polymorphic](Make encodeExpression polymorphic)
```
Comment (sjakobi) [inline dhall/CHANGELOG.md:20]:
```suggestion
* [Make `encodeExpression` polymorphic](https://github.com/dhall-lang/dhall-haskell/pull/1789)
```

File: dhall-yaml/CHANGELOG.md (hunk 2 above)
Comment (Gabriel439) [reply dhall-yaml/CHANGELOG.md:5]: Yeah, I keep forgetting that `dhall-json` contains `dhall-to-yaml` 🙂

**Comment by sjakobi** [comment]
https://github.com/dhall-lang/dhall-haskell/pull/1862 might be worth including in the release.

**Pull request merged**